website/src/components/Header/Header.js

```diff
@@ -1,16 +1,28 @@
-import React from 'react';
-import { Row, Col, Button } from 'react-bootstrap';
+import React from "react";
+import { Row, Col, Button, Jumbotron } from "react-bootstrap";
 
 export default function Heaser() {
   return (
     <Row className="text-center mb-5">
       <Col>
-        <h1>
-          This is BottlenoseMail!
-        </h1>
-
-        <Button href="#">Create New Inbox</Button>
+        <Jumbotron>
+          <Container>
+            <h1 className="display-4" style={{ color: "black" }}>
+              Bottlenose Mail
+            </h1>
+            <p className="lead" style={{ color: "black" }}>
```

Comment:
This is more of a style preference, but I like to include spaces in between major HTML elements.
```suggestion
            </h1>
            
            <p className="lead" style={{ color: "black" }}>
```